### ISSUE TITLE
[EPIC] Browser Plugin Connector

### ISSUE THREAD

**Issue by jrainville** [status-im/status-desktop#14794]
### Description

Connector between the Browser Plugin and the Status app.

#### Tasks

- [x] https://github.com/status-im/status-desktop/issues/14795

**Backend tickets**

- [x] https://github.com/status-im/status-desktop/issues/15179
- [x] https://github.com/status-im/status-desktop/issues/15182
- [x] https://github.com/status-im/status-desktop/issues/15197 
- [x] https://github.com/status-im/status-desktop/issues/15240
- [x] https://github.com/status-im/status-go/issues/5417
- [x] https://github.com/status-im/status-go/issues/5436
- [x] https://github.com/status-im/status-desktop/issues/15347
- [x] https://github.com/status-im/status-desktop/issues/15436
- [x] #15456
- [x] https://github.com/status-im/status-go/issues/5575
- [x] https://github.com/status-im/status-go/issues/5565
- [x] https://github.com/status-im/status-go/issues/5583
- [x] https://github.com/status-im/status-go/issues/5599
- [x] https://github.com/status-im/status-go/issues/5587
- [x] #15879
- [x] #15885
- [x] #15962
- [ ] https://github.com/status-im/status-go/issues/5528
- [ ] #16014
- [ ] #16015
- [x] #16063
- [ ] #16107

**Frontend tickets**

- [x] https://github.com/status-im/status-desktop/issues/15171
  -  Blocked by https://github.com/status-im/status-desktop/issues/14688
- [x] https://github.com/status-im/status-desktop/issues/15172 
  - Blocked by https://github.com/status-im/status-desktop/issues/14762
- [x] ~~https://github.com/status-im/status-desktop/issues/15173~~
  - Duplicate of https://github.com/status-im/status-desktop/issues/14759

(remaining issues will be added when the spike will be completed)

**Comment by noeliaSD** [comment]
We need to sync this EPIC with this one: https://github.com/status-im/status-desktop/issues/14588

I see some potential duplications here @jrainville

**Comment by iurimatias** [comment]
hey @felicio  this was marked as done in our board, so I was just making it consistent. currently we're doing a clean up and might move those remaining issues to the new github sub issues, I'll reopen this issue
